Multisig/UI/Settings/OwnerKeyManagement/OwnerKeyDetailsViewController/OwnerKeyDetailsViewController.swift

```diff
@@ -161,7 +161,7 @@ class OwnerKeyDetailsViewController: UITableViewController, WebConnectionObserve
 
         if App.shared.auth.isPasscodeSetAndAvailable {
             let vc = EnterPasscodeViewController()
-            vc.usesBiometry = false 
```

Comment:
does it just adds a space at the end of the line?

Please make sure your setting in Xcode set to auto-trim the lines 😄 
![image](https://user-images.githubusercontent.com/284056/164003472-dafb89b3-2727-4986-92fc-6f945a01fecc.png)

Reply:
As I understand it, it remove the trailing spce in that line.

Reply:
Mine looks exactly the same as yours: 
![image](https://user-images.githubusercontent.com/246473/164010116-57b3e31f-29f2-40ed-a6cd-08d7766e52a9.png)